src/test/java/StreamApiTest/GetAverageBalanceByBirthYearTest.java

```diff
@@ -0,0 +1,27 @@
+package StreamApiTest;
+
+import org.example.homework1.AccountService;
+import org.junit.Test;
+
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+
+public class GetAverageBalanceByBirthYearTest {
```

Comment:
Tests looks good, but you need only one class for all of them, you are testing AccountService